finder-app/writer.c

```diff
@@ -0,0 +1,41 @@
+#include <stdio.h>
+#include <syslog.h>
+#include <string.h>
+int main(int argc,char **argv)
+{
+	openlog(NULL,LOG_ODELAY,LOG_USER);
+	if(argc!=3)
+	{
+		printf("Requires two arguments. \n");
+		syslog(LOG_ERR,"Requires two arguments");
+		return 1;
+	}
+
+	char *filepath = argv[1];
+	char *str = argv[2];
```

Comment:
@lnxblog suggestion here: Use a const char* to the argument with descriptive name.